histolab/util.py

```diff
@@ -38,25 +38,24 @@ def apply_mask_image(img: PIL.Image.Image, mask: np.ndarray) -> PIL.Image.Image:
     Parameters
     ----------
     img : PIL.Image.Image
-        Input image
+            Input image
```

Comment:
could you explain the rationale of all this whitespaces added in the whole module? Thanks

Reply:
same, the [numpy docstring style document](https://numpydoc.readthedocs.io/en/latest/format.html) uses 1 tab / 4 spaces of indentation for the description with respect to the parameter name